Empire - The Last Battle/Assets/Unity/Scripts/MonoBehaviours/Inn.cs

```diff
@@ -0,0 +1,21 @@
+﻿using UnityEngine;
+using System.Collections;
+using System.Collections.Generic;
+
+public class Inn : MonoBehaviour
```

Comment:
Doesn't need to be another class, just stick it alongside cave in the overworld.